v1.10.1 to releases (#6661)
filecoin-project/lotus: CHANGELOG.md

# Lotus changelog

# 1.10.1 / 2021-07-05

> This is an optional but **highly recommended** release of Lotus that has many bug fixes and improvements based on the feedback we got from the community since HyperDrive.

Reviewer: ```suggestion
This is an optional but **highly recommended** release of Lotus for lotus miners that has many bug fixes and improvements based on the feedback we got from the community since HyperDrive.
```

> This is an optional but **highly recommended** release of Lotus that have many bug fixes and improvements based on the feedbacks we got from the community since HyperDrive.

Reviewer 2: ```suggestion
This is an optional but **highly recommended** release of Lotus that has many bug fixes and improvements based on the feedback we got from the community since HyperDrive.
```

Reviewer 2: Is it fair to say this release is only recommend for miners?

Reviewer: Yes it is

## New Features
- commit batch: AggregateAboveBaseFee config #6650

>   - `AggregateAboveBaseFee` is added to miner sealing configuration for setting the network base fee to start aggregating proofs. When the network base fee is lower than this value, the prove commits will be submitted individually via `ProveCommitSector`. According to the [Batch Incentive Alignment](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0013. md#batch-incentive-alignment) introduced in FIP-0013, we recommend miners to set this value to 0.15 nanoFIL(which is the default value) to avoid unexpected aggregation fee in burn and enjoy the most benefits of aggregation!

Reviewer 2: ```suggestion
  - `AggregateAboveBaseFee` is added to miner sealing configuration for setting the network base fee to start aggregating proofs. When the network base fee is lower than this value, the prove commits will be submitted individually via `ProveCommitSector`. According to the [Batch Incentive Alignment](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0013.md#batch-incentive-alignment) introduced in FIP-0013, we recommend miners to set this value to 0.15 nanoFIL(which is the default value) to avoid unexpected aggregation fee in burn and enjoy the most benefits of aggregation!
```
